ontohgis.ttl

```diff
@@ -17282,6 +16915,7 @@ ontohgis:administrative_system_7 rdf:type owl:NamedIndividual ,
                                  ontohgis:individualHasExternalIdentifier 7 ;
                                  ontohgis:startsDominationAt "1815-01-01"^^xsd:date ;
                                  dc:creator ontohgis:person_11 ;
+                                 <http://purl.org/dc/terms/isPartOf> ontohgis:administrative_system_49 ;
```

Comment:
I would use P46i_forms_part_of instead of http://purl.org/dc/terms/isPartOf because the latter is an annotation property and we need an object property here.

Reply:
Ok. Agree